bridgev2/bridgeconfig/config.go

```diff
@@ -50,7 +50,8 @@ type ProvisioningConfig struct {
 }
 
 type DirectMediaConfig struct {
-	Enabled                bool `yaml:"enabled"`
+	Enabled                bool   `yaml:"enabled"`
+	MediaIDPrefix          string `yaml:"media_id_prefix" json:"media_id_prefix"`
```

Comment:
Bridge configs are yaml only so this doesn't need the json tag

```suggestion
	MediaIDPrefix          string `yaml:"media_id_prefix"`
```